exampleSite/config.toml

```diff
@@ -221,10 +221,16 @@ enableEmoji = true
         Matrix = ""
         Bilibili = ""
         Email = "xxxx@xxxx.com"
+        Phone = "555-555-555"
+        Location = "39.9047/116.3480"
         RSS = true
         [languages.en.params.social.Mastodon]
           id = "@xxxx"
           prefix = "https://mastodon.technology/"
+        [languages.en.params.social.pgp]
+          id = "0xAAAAAAAAAAAAAAAA"
+          # Point to any keyserver or your site
+          prefix = "https://example.com/keys/"
```

Comment:
Still feels wrong. Sorry for nitpicking, just want to hone it to perfection, you know ;) You may ignore my comments, it's up to @dillonzq to merge or decline the request anyway.

That being said, why have prefix and ID here at all? Why not a single field, just with no pre-filled prefix?

No one in their right mind would save their key as `static/keys/0xDEADBEEF`, it would likely just be `static/mykey.asc`.

I suggest we don't add a section to `config.toml`, just a PGP parameter, and no prefix field in `social.yml`. Then the user can either
```
PGP = "/pgpkey.asc"
```
for a local key saved in `/static`, or
```
PGP = "https://pgp.key-server.io/0xDEADBEEF"
```
for a key on a third-party server. Of course, the examples should be in sample `config.toml` as comments.

Reply:
You are right.